### PR TITLE
Bug 1225765 - [Gallery] Port crop aspect ratio to use gaia-menu

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1225765

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2cc7cca520cc0487d6970eeb287b63a57dae1bf6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2cc7cca520cc0487d6970eeb287b63a57dae1bf6

File: apps/gallery/index.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
     <link rel="stylesheet" type="text/css" href="shared/style/status.css">
     <!-- These file are loaded lazily but listed here to get packaged
     <link rel="stylesheet" href="shared/style/confirm.css" type="text/css" />
-    <link rel="stylesheet" type="text/css" href="shared/style/action_menu.css" />
+    <script defer src="shared/js/component_utils.js"></script>
+    <script defer src="shared/elements/gaia_menu/script.js"></script>
     -->
```
Comment (punamdahiya) [inline apps/gallery/index.html]: Thanks for ensuring we lazy load these files!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cbefeace5ca82cfc0a8dfd527acc6aa1203ed70a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=28f1358ddaf389200191af173fba49110c7b0604

**Pull request merged**